src/main/kotlin/com/cjbooms/fabrikt/cli/CodeGenOptions.kt

```diff
@@ -38,6 +38,10 @@ enum class ClientCodeGenTargetType(val description: String) {
     OPEN_FEIGN("Generate OpenFeign client.");
 
     override fun toString() = "`${super.toString()}` - $description"
+
```

Comment:
I'm guessing this is just an unrelated refactor? I have no issue with it either way.

Reply:
Yes, this is only to allow referencing the default setting from both the CLI and the Playground.